presentation/frontend/src/presentation-frontend/favorite-properties/FavoritePropertiesStorage.ts

```diff
@@ -156,74 +140,62 @@ export class OfflineCachingFavoritePropertiesStorage implements IFavoritePropert
       // note: we're copying the cached values to temp arrays because `saveProperties` and `savePropertiesOrder` both
       // attempt to modify cache dictionaries
 
-      // SWB
-      const propertiesCache = new Array<{ properties: Set<PropertyFullName>, projectId?: string, imodelId?: string }>();
-      // SWB
-      this._propertiesOfflineCache.forEach((key, value) => propertiesCache.push({ properties: value, projectId: key[0], imodelId: key[1] }));
-      // SWB
-      propertiesCache.forEach(async (cached) => this.saveProperties(cached.properties, cached.projectId, cached.imodelId));
+      const propertiesCache = new Array<{ properties: Set<PropertyFullName>, iTwinID?: string, imodelId?: string }>();

```

Comment:
Spelling error with iTwinID

Reply:
Fixed the iTwinID misspelling, but the second question persists

Reply:
I'd change it to `iTwinId` and `iModelId` if we can without breaking too many things...

Reply:
Is that convention documented anywhere? I personally like `itwinId` and `imodelId` better than `iTwinId` and `iModelId` and we use that convention across presentation packages.

Reply:
As someone who has had to type `iTwin` hundreds of times this week, your first point is very true. I am not aware of any explicit documentation about this but my best counter arguments are:
* The official terms publicized terms are `iTwin` and `iModel`
* `iModel` is currently used much more frequently than `imodel`

Reply:
A decision was made to make all non-local names conform to using a capital `M` or `T` and to capitalize the `i` according the the camel or pascal-casing.
i.e: `iModelName` or `namedIModel` for camel-case such as parameters and `ITwinClient` or `AuthorizedITwin` for pascal-case such as interface names.

However, correcting past uses is beyond the scope of this PR and will be done in the future.